lib/si-frontend-types-rs/src/audit_log.rs

```diff
@@ -0,0 +1,21 @@
+use serde::{Deserialize, Serialize};
+use si_events::{
+    audit_log::{AuditLogKind, AuditLogService},
+    Actor,
+};
+
+#[derive(Clone, Debug, Deserialize, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct AuditLog {
```

Comment:
This the "frontend type" of the audit log. Note that this is explicitly not the backend representation, which should represent the data model of the audit log in the database. That is not included in this PR since this PR is aimed at preparing the frontend for mocked data.